src/game_logic/entity_configuration.ipp

```diff
@@ -203,13 +203,13 @@ Message MESSAGE_TYPE_BY_INDEX[] = {
 
 int messengerDroneTypeIndex(const ActorID id) {
   switch (id) {
-    case 213:
-    case 214:
-    case 215:
-    case 216:
-      return id - 213; // 0 to 3
+    case ActorID::Messenger_drone_YOUR_BRAIN_IS_OURS:
+    case ActorID::Messenger_drone_BRING_BACK_THE_BRAIN:
+    case ActorID::Messenger_drone_LIVE_FROM_RIGEL_ITS_SATURDAY_NIGHT:
+    case ActorID::Messenger_drone_DIE:
+      return static_cast<int>(id) - 213; // 0 to 3
```

Comment:
From my point of view the cast is a little bit "hacky", but couldn't think of a better solution so far.

Reply:
Agreed, writing out `return 0`, `return 1` etc. for the different cases is probably cleaner and easier to read.